packages/cutler/bin/cutler.dart

```diff
@@ -27,24 +30,60 @@ class Cutler extends CommandRunner<int> {
       ..addFlag('update', defaultsTo: true, help: 'Update checkouts.');
   }
 
+  final Logger _logger;
+
+  Iterable<String> missingDirectories(String rootDir) {
```

Comment:
A docstring would be helpful here. "Returns paths to all missing repositories (see [Repo.values])" or something